### ISSUE TITLE
TS map containment 

### ISSUE THREAD

**Issue by israelmcmc** [cositools/cosipy#137]
From @StevenBoggs https://ucbssl.slack.com/archives/C02TWC7U4NL/p1708103870520689?thread_ts=1708039678.416039&cid=C02TWC7U4NL

another issue I seem to be having with the GRB imaging notebooks is that all of the sky plots seem to be 90% containment whether the containment is defined as 0.9 or not. i.e., When I do the subsequent ts.plot_ts commands the images are identical, and both are labeled "Containment 90.0%". Perhaps the containment parameter needs to be reset between calls?

**Comment by Yong2Sheng** [comment]
Hi @israelmcmc and @StevenBoggs:

There is a bug that plots the default TS map with a 90% containment level, so the row TS map looks the same as the 90% one. It has now been fixed.

The different containment levels produce visually identical TS maps for the GRB with full signal and the Crab cases. I think it is because the signals are too strong for these two cases. I added text in the notebook to caution the users about this.

I will make a PR to reflect the fixes.

Thanks!
